File: src/randsim.cpp
#include <iostream>
#include <string>

int main(int argc, char* argv[]) {
    int rl = std::stoi(std::string(argv[1]));
    double td = std::stod(std::string(argv[2]));
    double theta = std::stod(std::string(argv[3]));
    std::string genome = argv[4];
    std::string out_stem = argv[5];

    std::cout << "read_len = " << rl << "\n";
    std::cout << "target_depth = " << td << "\n";
    std::cout << "theta = " << theta << "\n";
    std::cout << "genome = " << genome << "\n";
    std::cout << "out_stem = " << out_stem << "\n";
    return 0;
}

File: src/RandSim.java
public class RandSim {
        public static void main(String[] args) {
          int rl = Integer.parseInt(args[0]);
          double td = Double.parseDouble(args[1]);
          double theta = Double.parseDouble(args[2]);
          String genome = args[3];
          String out_stem = args[4];

          System.out.format("read length: %d\n", rl);
          System.out.format("target depth: %f\n", td);
          System.out.format("theta : %f\n", theta);
          System.out.format("genome : %s\n", genome);
          System.out.format("out_stem : %s\n", out_stem);
        }
}